core-services/prow/02_config/opendatahub-io/odh-manifests/_pluginconfig.yaml

```diff
@@ -38,6 +38,7 @@ plugins:
     - retitle
     - shrug
     - trigger
+    - transfer-issue
```

Comment:
If we are deprecating odh-manifests, I would not worry about adding this plugin since we can do this manually in github (or the gh cli) and disable GitHub issues in odh-manifests https://github.com/opendatahub-io/opendatahub-community/issues/77

Reply:
i will revert this change.